include/seastar/core/reactor.hh

```diff
@@ -148,6 +150,67 @@ public:
     virtual void set_exception(std::exception_ptr eptr) noexcept = 0;
 };
 
+struct disk_params {
```

Comment:
reactor.hh is the wrong place to put this into. Please, add a header under include/seastar/core/internal/ and place both, disk_params and disk_config_params under internal namespace within it